Dockerfile

```diff
@@ -1,29 +1,31 @@
-FROM node:10.13.0
+FROM node:12.14-alpine
 
 WORKDIR /app
 
-# Set up deploy user and working directory
-RUN apt-get update && apt-get install -y \
+RUN apk --no-cache --quiet add \
+  build-base \
   dumb-init \
-  libgpm2 \
-  libslang2 \
-  mc \
-  mc-data \
-  unzip && \
-  adduser --disabled-password --gecos '' deploy
-
-# Install the packages
-COPY package.json yarn.lock ./
-COPY patches ./patches
-RUN yarn install
+  git \
+  python && \
```

Comment:
Why do we need Python in the app image?

Reply:
This is required by the `appmetrics` package, which seems to be doing some node profiling.